### ISSUE TITLE
change how we do rights statements

### ISSUE THREAD

**Issue by apinkney0696** [sciencehistory/scihist_digicoll#1634]
In our most recent meetings about Video, we brainstormed ways to maximize the clarity of the Institute's copyright policy. The solution we came up with, which has been reported to and supported by Michelle and Gabriela, is adding a tooltip next to the large Rights Statement, beneath the representative image.

The tooltip, something like this ![image](https://user-images.githubusercontent.com/78036563/161065233-ab41b232-8847-4877-b71c-bc83efc5b400.png), would lead users directly to the Copyright section on the About page. 

Here is the idea sketched out:
![IMG_2647](https://user-images.githubusercontent.com/78036563/161065703-f7946ca9-537a-4275-ac4e-6cec4069f05c.jpeg)

In summary, this request is to:
1. Add a tooltip next to the large copyright statement under the representative image. A question mark icon for this would be best.
2. Hovering over the tooltip should bring up a small textbox that says "What is this?"
3. Clicking on the tooltip should take a user directly to the Copyright section of the About page.

**Comment by jrochkind** [comment]
Hi!

Is this just for the video page, or for all pages? 

i'd be inclined to do it on all pages, for consistency?

Is it just for certain copyright status, or all statuses? 

I'm not sure if the "hover to bring up a small textbox that says "What is this?" is a beset pracitce/standard UI/UX:
* For accessibility, we need to support keyboard navigation -- But when you keyboard navigate to activate that element, should it trigger the "hover" message, or the link? Probably the link?  
* Keep in mind that "hover" actions are not triggerable on mobile/touch devices
* I'm not sure if the standard utilities we're using (from bootstrap) for tooltips will support one action on hover and another action on click, but we can investigate. There might be a simple way to do it. 
* Do you have any examples of websites with UI elements like this, that we can use as a model?

**Comment by apinkney0696** [comment]
Based on @jrochkind and my meeting yesterday, we are re-working the copyright information plan as follows:

The copyright statement link located beneath the representative image and the copyright statement link in the metadata display will no longer link to rightstatement.org but instead will send the user to a new page (or a pop-up window?). On this page, there will be a section that includes the paragraph of information about the rights statement from rightstatement.org AND our institutional language around copyright.

There will be one page per copyright statement, excluding Public Domain and CC licensed works, which will continue to link to the Creative Commons website.
![thumbnail_image001](https://user-images.githubusercontent.com/78036563/171671345-dc3b8578-85c1-4379-942d-2a8ef232dfb4.png)

Let me know if this captures the plan sufficiently, Jonathan!

**Comment by jrochkind** [comment]
@apinkney0696 I'm going to go ahead and change the link on "U.S. Copyright Office website" from https://www.copyright.gov/ to https://www.copyright.gov/fair-use/more-info.html , unless you tell me not to!

**Comment by jrochkind** [comment]
@apinkney0696 As I make sure our local explanation has everything it needs to from rightsstatements.org, looking at the bullet points that rightsstatements.org provides under "Notices", most of them seem to be taken care of by our own specific instructions, but there's this one, that might make sense to add to our own copyright explanation?

> You may need to obtain other permissions for your intended use. For example, other rights such as publicity, privacy or moral rights may limit how you may use the material.

I am going to experimentally add that to my draft page, making a note here so we make sure to review it and decide if it's a good idea.

**Comment by apinkney0696** [comment]
That's fine about replacing the link @jrochkind! Thank you!

And that sounds great about adding the sentence to the draft.

**Comment by apinkney0696** [comment]
Finished?
